lib/videojs-playlist-ui.vertical.less

```diff
@@ -0,0 +1,175 @@
+    height: 100%;
+  }
+
+  .vjs-playlist-duration {
+    background-color: rgba(26, 26, 26, 0.8);
+    left: 0;
+    padding: 2px 8px;
+    position: absolute;
+    top: 0;
+  }
+  .vjs-selected .vjs-playlist-duration {
+    display: none;
+  }
+
+  cite {
+    display: block;
+    font-style: normal;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
+  .vjs-playlist-description {
+    color: @secondary-color;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    margin: 0;
+    white-space: nowrap;
+  }
+}
+
+/* Prevent interaction with the playlist menu while ads are playing */
+/* on browsers that don't support pointer-events (IE<11), prevent
+scrolling past the ad overlay */
+.vjs-playlist.vjs-ad-playing {
+  overflow: hidden;
+}
+/* prevent clicks and scrolling from affecting the playlist during ads */
+.vjs-playlist.vjs-ad-playing.vjs-csspointerevents {
+  pointer-events: none;
+  overflow: auto;
+}
+/* darken the playlist menu display to indicate it's not interactive
+during ads and capture click events on IE<11 */
+.vjs-playlist.vjs-ad-playing .vjs-playlist-ad-overlay {
+  display: none;
+}
+.vjs-playlist.vjs-ad-playing .vjs-playlist-ad-overlay {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  /* IE8 fallback */
+  background-color: #1a1a1a;
+  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
+
+  /* modern browsers */
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+/* Parametric rules. These are specialized for touch and mouse-based devices */
+.playlist-base(@base-font-size: 14px) {
+  @small-font-size: ceil(@base-font-size * 0.75);
+  @playlist-padding: @base-font-size;
+  @item-height: @base-font-size * 4;
+  @thumbnail-width: ceil(@item-height * (16/9));
+  @name-height: @base-font-size;
+  @description-height: @item-height - @name-height;
+
+  font-size: @base-font-size;
+  min-width: @base-font-size * 17;
+  background-color: @background-color;
+  color: @main-color;
+  padding: @playlist-padding @playlist-padding 0 @playlist-padding;
+
+  .vjs-playlist-item {
+    height: @item-height;
+    margin-bottom: @playlist-padding;
+  }
+
+  .vjs-playlist-thumbnail {
+    height: @item-height;
+    font-size: @small-font-size;
+    margin-right: @playlist-padding;
+    width: @thumbnail-width;
+  }
+  .vjs-selected .vjs-playlist-thumbnail::before {
+    height: @item-height;
+    line-height: @item-height;
+    width: @thumbnail-width;
+  }
+
+  time {
+    font-size: @small-font-size;
+  }
+
+  cite {
+    height: @name-height;
+    line-height: @name-height;
+  }
+
+  .vjs-playlist-description {
+    height: @description-height;
+    line-height: ceil(@base-font-size * 1.5);
+  }
+}
+
+/* Touch-device playlist dimensions */
+.vjs-playlist {
+  .playlist-base;
+}
+
+/* Mouse-only playlist dimensions */
+.vjs-mouse.vjs-playlist {
+  .playlist-base(17px);
+
+  .vjs-playlist-item:hover .vjs-playlist-thumbnail::before {
```

Comment:
All for autoprefixing. I thought the only usecase we had for postcss was removing the extra `:` in pseudo elements.